src/components/views/messages/DecryptionFailureBody.tsx

```diff
@@ -33,15 +33,33 @@ function getErrorMessage(mxEvent: MatrixEvent, isVerified: boolean | undefined):
 
         case DecryptionFailureCode.HISTORICAL_MESSAGE_USER_NOT_JOINED:
             return _t("timeline|decryption_failure|historical_event_user_not_joined");
+
+        case DecryptionFailureCode.SENDER_IDENTITY_PREVIOUSLY_VERIFIED:
+            return _t("timeline|decryption_failure|sender_identity_previously_verified");
+
+        case DecryptionFailureCode.UNSIGNED_SENDER_DEVICE:
+            // TODO: event should be hidden instead of showing this error (only
```

Comment:
I think this needs to be done by checking the decryption failure code in `EventTile.tsx`, and if it is this code, then render an empty tile?